src/core/player.h

```diff
@@ -119,6 +120,14 @@ class Player : public PlayerInterface {
   explicit Player(Application* app, QObject* parent = nullptr);
   ~Player();
 
+  static const char* kSettingsGroup;
+
+  // Don't change the values
```

Comment:
I believe it was written like this somewhere else in the code, but that is indeed the reason. I will clarify it